PVSSpringBoot/src/main/java/com/example/PVSSpringBoot/ControllerPackage/ProductController.java

```diff
@@ -9,8 +9,9 @@
 
 @RestController
 @RequestMapping("/products")
+@CrossOrigin
 public class ProductController {
-
+    @Autowired
```

Comment:
the annotation of @Autowired already exists on top of the class constructor, I don't think you have to add it here too.